pom.xml

```diff
@@ -90,9 +90,4 @@
     </site>
   </distributionManagement>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
The 2.30+ parent should be fine for older cores; as far back as 1.580.1 (implying a Java 6 baseline) is definitely supported.